helm/frost-server/templates/mqtt-deployment.yaml

```diff
@@ -64,15 +64,15 @@ spec:
             - name: mqtt_Enabled
               value: "true"
             - name: mqtt_Port
-              value: "1883"
```

Comment:
This last port shouldn't be changed to `.Values.frost.mqtt.ports.mqtt.servicePort`, same reason as the others.

Reply:
My bad, it is fixed now